### DISCUSSION TITLE
bug: Go lsp loading error

### DISCUSSION THREAD

**Discussion by dotfrag** [LazyVim/LazyVim#3381] [Q&A] [answered]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.10

### Operating system/version

Linux

### Describe the bug

When I open any of the go projects on my pc, go lsp loads fine. However, when I open any go file from the lazygit repo, I get this error three times:

```text
Error  06:02:44 PM lsp.message LSP Message (gopls) Invalid settings: parsing setting "gofumpt": duplicate configuration for gofumpt
Error  06:02:45 PM lsp.message LSP Message (gopls) Invalid settings: parsing setting "gofumpt": duplicate configuration for gofumpt                                 
Error  06:02:45 PM lsp.message LSP Message (gopls) Invalid settings: parsing setting "gofumpt": duplicate configuration for gofumpt
```

Why 3 times? Is it trying to load the project 3 times? I don't see anything unusual in the repository that would cause this.

### Steps To Reproduce

1. Use LazyVim (starter)
2. `git clone --depth 1 https://github.com/jesseduffield/lazygit`
3. `cd lazygit`
4. `nvim main.go`

### Expected Behavior

Expected to not have errors.

### Repro

LazyVim starter with go extra enabled.

**Comment by dpetka2001** [comment, marked as answer]
Probably because of the `.vscode/settings.json`. There are duplicate entries about `gofumpt`. Not sure what the correct approach for this is. I'm guessing it's because `neoconf.nvim` is configured to import `.vscode/settings.json` by default. You could add a `.lazy.lua` file at the root of the repo with the following contents
```lua
return {
	{
		"folke/neoconf.nvim",
		opts = {
			import = {
				vscode = false,
			},
		},
	},
}
```
And you could just not track this file when you want to push something.

**Reply by dotfrag** [reply]
> I'm guessing it's because neoconf.nvim is configured to import .vscode/settings.json by default.

I wasn't aware of this, thanks for the info!
